model/manifest.go

```diff
@@ -0,0 +1,102 @@
+package model
```

Comment:
What is the reason we can't vendor `fissile` to get this functionality instead? Can we do some simple change to `fissile` to make this possible?

Reply:
:+1:  I'm going to make a small test project to see what's happening.